generators/getMetricTestGenerator.js

```diff
@@ -3,7 +3,7 @@ const fs = require('fs')
 const fsExtra = require('fs-extra')
 const functionsList = require('../utils/functionsList')
 
-const LIST_OF_DAILY_METRICS = [
+const LIST_OF_IGNORED_METRICS = [
```

Comment:
Are you sure that these metrics need to be ignored? And what ignore means in this context?

Reply:
That the test generator doesn't automatically generate a test for them, I do it manually. I'm kind of thinking about removing the whole thing, starting to think that you were right about this when I was first introducing it.